### PR TITLE
Rake task

### PR DESCRIPTION
**Pull request by nathangthomas**
This PR does the following: 
- Adds RtdLocation model
- Adds RtdLocation table to database
- Adds service to consume shred share microservice data
- Adds creator poro to create rtd location objects
- Adds rake task to import rtd location objects to data base 
(run rake task with => `rake import:rtd`)
- Adds background image to app

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -52,9 +52,13 @@ body
 {
   margin: 0;
   padding: 0;
+  background-image: url("http://clipart-library.com/images_k/mountain-transparent/mountain-transparent-12.png");
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-size: 100%;
```
Comment (leiyakenney) [inline app/assets/stylesheets/application.css.scss:58]: Yessss can't wait for this background!

File: app/models/rtd_location_creator.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class RtdLocationCreator
+
+  def create_record(rtd_locations_data)
+    parsed_data = JSON.parse(rtd_locations_data)
+    parsed_data.map do |data|
+      data[1].map do |location|
+        parsed_data = JSON.parse(location, symbolize_names: true)
+
+        city = data[0]
+        name = parsed_data[:name]
```
Comment (leiyakenney) [inline app/models/rtd_location_creator.rb:10]: Kelly mentioned briefly to me about how the city is called differently than the actual location's data. Great work on figuring that out!!

File: app/services/rtd_locations_service.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class RtdLocationsService
+
+  def self.rtd_locations
+    conn = Faraday.new(:url => 'https://shred-share-api.herokuapp.com')
+    conn.get '/rtd_locations/index'
+  end
```
Comment (leiyakenney) [inline app/services/rtd_locations_service.rb:6]: Nice abstraction!

File: lib/tasks/rtd_location_create.rake [hunk 4]
```diff
@@ -0,0 +1,14 @@
+require 'yaml'
+
```
Comment (leiyakenney) [inline lib/tasks/rtd_location_create.rake:2]: This is new! What does it do?

File: lib/tasks/rtd_location_create.rake [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+   task rtd: :environment do
+
+     location_data = RtdLocationsService.rtd_locations
+
+     creator= RtdLocationCreator.new
+     creator.create_record(location_data.body)
+
+     p "Import Successful!"
+   end
+ end
```
Comment (leiyakenney) [inline lib/tasks/rtd_location_create.rake:14]: Actually, I'd love to walk through this whole file, since I'm not familiar with rake

**Review by leiyakenney — APPROVED** [review]

**Comment by hsmitha26** [comment]
Nice work!

File: lib/tasks/rtd_location_create.rake (hunk 4 above)
Comment (nathangthomas) [reply lib/tasks/rtd_location_create.rake:2]: Actually we can delete that. It reads the application.yml file so the rake task knows about my API keys, but we aren't' doing that in our rails app.  It is habit for me to include this at the top of my rake tasks so I'm not confused when my rake does not work.

**Comment by hsmitha26** [comment]
Reviewed as a group.

**Pull request merged**